app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Api::V1::BaseController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  protect_from_forgery with: :null_session
+
+  def render_success(message, data)
+    render json: {
+        status: 'success',
+        message: message,
+        data: data
+    }
+  end
+
+  def render_error(errors, data)
+    render json: {
+        status: 'error',
+        data: data,
+        errors: errors
+    }, status: 422
```

Comment:
Let's pass default param to data attribute, 
Like : 
def render_error(errors, data = []) {
}